src/rideables/MontageLfHashTable.hpp

```diff
@@ -95,8 +95,16 @@ class MontageLfHashTable : public RMap<K,V>, public Recoverable{
     }
 public:
     MontageLfHashTable(GlobalTestConfig* gtc) : Recoverable(gtc), tracker(gtc->task_num, 100, 1000, true), gtc(gtc) {
+        if (get_recovered_pblks()) {
+            recover();
+        }
+    };
+    ~MontageLfHashTable(){
```

Comment:
Why is this nontrivial destructor necessary _for MontageLfHashTable_? If it's indeed necessary, would other Montage data structures need this as well, i.e., MontageHashTable and MontageGraph?